sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -27,13 +27,17 @@ def initialize
         @text_injectors = nil
         @span_processors = []
         @use_mode = USE_MODE_UNSPECIFIED
-        @tracer_provider = Trace::TracerProvider.new
+        @resource = Resources::Resource.telemetry_sdk
       end
 
       def logger
         @logger ||= Logger.new(STDOUT)
       end
 
+      def resource=(new_resource)
+        @resource = @resource.merge(new_resource)
```

Comment:
Do we want to `merge` here or just replace? In either case, we should document the behavior.

Reply:
We definitely want `merge`, since we always want to have the default SDK resource labels. So this is just a request to document the method.

Reply:
https://github.com/open-telemetry/opentelemetry-ruby/pull/263/commits/ec14e075b254d72f2eca2284a2548c79de45b59d#diff-938e8162d01853e91a23d9ab6d45636aR37-R41